FirebaseUI/Database/API/FirebaseArrayDelegate.h

```diff
@@ -36,7 +38,7 @@
  * @param object The object added to the FirebaseArray
  * @param index The index the child was added at
  */
-- (void)childAdded:(id)object atIndex:(NSUInteger)index;
+- (void)array:(FirebaseArray *)array didAddObject:(id)object atIndex:(NSUInteger)index;
```

Comment:
FWIW we consider FirebaseArray to be a "private API" even though some are accessing it.  So we can flag it in the changelog but we don't have to worry too much.

On Android it's not possible to use FirebaseArray directly due to the access modifiers.